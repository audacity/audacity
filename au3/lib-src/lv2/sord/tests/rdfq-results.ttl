# from http://www.w3.org/2003/03/rdfqr-tests/recording-query-results.html

@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix rs:     <http://jena.hpl.hp.com/2003/03/result-set#> .


<>  rdf:type rs:ResultSet ;
    rs:size 4 ;
    rs:resultVariable "x" ; rs:resultVariable "y" ;
    rs:solution
        [ rdf:type rs:ResultSolution ;
          rs:binding [ rs:variable "x" ; rs:value 123 ] ;
          rs:binding [ rs:variable "y" ; rs:value <http://example.com/resource1> ]
        ] ;

    rs:solution
        [ rdf:type rs:ResultSolution ;
          rs:binding [ rs:variable "x" ;
                      rs:value "2003-01-21" ] ;
          rs:binding [ rs:variable "y" ;
                      rs:value <http://example.com/resource2> ]
        ] ;

    rs:solution
        [ rdf:type rs:ResultSolution ;
          rs:binding [ rs:variable "x" ;
                      rs:value "anon1" ] ;
          rs:binding [ rs:variable "y" ;
                      rs:value _:a ]
        ] ;

    rs:solution
        [ rdf:type rs:ResultSolution ;
          rs:binding [ rs:variable "x" ;
                      rs:value "anon2" ] ;
          rs:binding [ rs:variable "y" ;
                      rs:value _:a ]
        ] ;
    .
