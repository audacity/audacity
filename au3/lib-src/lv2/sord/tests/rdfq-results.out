_:genid1 <http://jena.hpl.hp.com/2003/03/result-set#variable> "x" .
_:genid1 <http://jena.hpl.hp.com/2003/03/result-set#value> "123"^^<http://www.w3.org/2001/XMLSchema#integer> .
_:genid2 <http://jena.hpl.hp.com/2003/03/result-set#variable> "y" .
_:genid2 <http://jena.hpl.hp.com/2003/03/result-set#value> <http://example.com/resource1> .
_:genid3 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://jena.hpl.hp.com/2003/03/result-set#ResultSolution> .
_:genid3 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid1 .
_:genid3 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid2 .
_:genid4 <http://jena.hpl.hp.com/2003/03/result-set#variable> "x" .
_:genid4 <http://jena.hpl.hp.com/2003/03/result-set#value> "2003-01-21" .
_:genid5 <http://jena.hpl.hp.com/2003/03/result-set#variable> "y" .
_:genid5 <http://jena.hpl.hp.com/2003/03/result-set#value> <http://example.com/resource2> .
_:genid6 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://jena.hpl.hp.com/2003/03/result-set#ResultSolution> .
_:genid6 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid4 .
_:genid6 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid5 .
_:genid7 <http://jena.hpl.hp.com/2003/03/result-set#variable> "x" .
_:genid7 <http://jena.hpl.hp.com/2003/03/result-set#value> "anon1" .
_:genid8 <http://jena.hpl.hp.com/2003/03/result-set#variable> "y" .
_:genid8 <http://jena.hpl.hp.com/2003/03/result-set#value> _:a .
_:genid9 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://jena.hpl.hp.com/2003/03/result-set#ResultSolution> .
_:genid9 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid7 .
_:genid9 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid8 .
_:genid10 <http://jena.hpl.hp.com/2003/03/result-set#variable> "x" .
_:genid10 <http://jena.hpl.hp.com/2003/03/result-set#value> "anon2" .
_:genid11 <http://jena.hpl.hp.com/2003/03/result-set#variable> "y" .
_:genid11 <http://jena.hpl.hp.com/2003/03/result-set#value> _:a .
_:genid12 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://jena.hpl.hp.com/2003/03/result-set#ResultSolution> .
_:genid12 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid10 .
_:genid12 <http://jena.hpl.hp.com/2003/03/result-set#binding> _:genid11 .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://jena.hpl.hp.com/2003/03/result-set#ResultSet> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#size> "4"^^<http://www.w3.org/2001/XMLSchema#integer> .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#resultVariable> "x" .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#resultVariable> "y" .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#solution> _:genid3 .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#solution> _:genid6 .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#solution> _:genid9 .
<http://www.w3.org/2001/sw/DataAccess/df1/tests/rdfq-results.ttl> <http://jena.hpl.hp.com/2003/03/result-set#solution> _:genid12 .
